packages/ember-svg-jar/addon/template-registry.d.ts

```diff
@@ -0,0 +1,6 @@
+import type SvgJar from './types';
+
+export default interface EmberSvgJarRegistry {
+  'svg-jar': typeof SvgJar;
+  svgJar: typeof SvgJar;
```

Comment:
Don't think this one is needed, as you can't use `{{svgJar}}` as a helper?

Reply:
correct